packages/demo-next/next-git-media-store.js

```diff
@@ -16,12 +16,30 @@ limitations under the License.
 
 */
 
-import { GitMediaStore } from '@tinacms/git-client'
+import { GitMediaStore, nextOffset } from '@tinacms/git-client'
 
 export class NextGitMediaStore extends GitMediaStore {
   previewSrc(src) {
     return /jpg|jpeg|png|svg|gif$/.test(src.toLowerCase())
       ? src.replace('/public', '')
       : null
   }
+  async list(options) {
```

Comment:
what if we just called `super.list` and filtered the results?

Reply:
good call!